config/initializers/smtp.rb

```diff
@@ -1,13 +1,20 @@
 if Rails.env.development?
-ActionMailer::Base.delivery_method = :smtp
-ActionMailer::Base.smtp_settings = {
-  :address              => "smtp.zoho.com",
-  :port                 => 465,
-  :user_name            => 'countries_launch_academy@zoho.com',
-  :password             => 'MakeNoBugs!',
-  :authentication       => :login,
-  :ssl                  => true,
-  :tls                  => true,
-  :enable_starttls_auto => true
+  Rails.application.config.action_mailer.delivery_method = :smtp
+  Rails.application.config.action_mailer.smtp_settings = {
+  :address => "127.0.0.1",
+  :port => 1025
 }
 end
+
+if Rails.env.production?
+  Rails.application.config.action_mailer.delivery_method = :smtp
+  Rails.application.config.action_mailer.smtp_settings = {
+    :address   => "smtp.mandrillapp.com",
+    :port      => 587, # ports 587 and 2525 are also supported with STARTTLS
+    :enable_starttls_auto => true, # detects and uses STARTTLS
+    :user_name => "peterrko1004@gmail.com",
+    :password  => "OqBNJ7bhZNHqhAqz2ZYV0w", # SMTP password is any valid API key
+    :authentication => 'login', # Mandrill supports 'plain' or 'login'
+    :domain => 'https://immense-oasis-2395.herokuapp.com/', # your domain to identify your server when connecting
```

Comment:
Line is too long. [113/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.